docs/pages/includes/role-spec.mdx

```diff
@@ -73,11 +79,11 @@ spec:
       # session_recording_config resource has set 'mode: off'.
       desktop: true
```

Comment:
A bit confusing that the comment above doesn't mention `true` as a valid option.

Reply:
I've added to the comment which values are available for the `desktop` protocol.